custom/icds_reports/static/js/directives/navigation/navigation.directive.js

```diff
@@ -26,10 +26,25 @@ function NavigationController($window, $scope, $route, $routeParams, $location,
     };
 
     $scope.goToStep = function(path, params) {
+
+        var selectedMonth = $location.search()['month'] !== void(0) ? parseInt($location.search()['month']) : new Date().getMonth() + 1;
+        var selectedYear = $location.search()['year'] !== void(0) ? parseInt($location.search()['year']) : new Date().getFullYear();
+
+        var isServiceDeliveryDashboard = path.indexOf('service_delivery_dashboard') !== -1;
+        if (isServiceDeliveryDashboard && selectedYear <= 2018) {
+            if (selectedYear < 2018 || (selectedYear === 2018 && selectedMonth === 1)) {
+                $location.search('month', new Date().getMonth() + 1);
+                $location.search('year', new Date().getFullYear());
+                $rootScope.dateChanged = true;
+                params = $location.search();
```

Comment:
Will it change the query param in the url also? or its just a ajax request?

Reply:
The params will be changed in both locations.